style.css

```diff
@@ -1,533 +1,627 @@
+  list-style-type: none;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  justify-content: flex-start;
+}
+
+.project-button,
+.resume-button {
+  background-color: #fff;
+  color: #6070ff;
+  font-weight: 500;
+  margin-top: 20px;
+  margin-left: 16px;
+  width: 35%;
+  height: 48px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+}
+
+.resume-button {
+  margin-left: 24px;
+}
+
+.about-me {
+  background: #fff;
+  margin-top: 88px;
+  border-top-right-radius: 80px;
+}
+
+.btn:hover {
+  cursor: pointer;
+  background-color: #6070ff;
+  color: #fff;
+}
+
+.languages {
+  margin-top: 30px;
+  margin-left: 24px;
+}
+
+.languages h3 {
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  margin-bottom: 30px;
+}
+
+.languages .vector {
+  cursor: pointer;
+  width: 20px;
+  height: 11px;
+}
+
+.languages-heading {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+  -webkit-box-align: baseline;
+  -ms-flex-align: baseline;
+  align-items: baseline;
+  margin-right: 24px;
+}
+
+.languages a {
+  text-decoration: none;
+  color: #000;
+}
+
+.language {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  background-color: #f7f7f9;
+  padding: 10px;
+  margin-right: 30px;
+  margin-bottom: 0;
+}
+
+.language-text {
+  margin-left: 12px;
+  -ms-flex-item-align: center;
+  -ms-grid-row-align: center;
+  align-self: center;
+}
+
+.language img {
+  width: 48px;
+}
+
+.language h4 {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.frameworks {
+  margin-top: 30px;
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.frameworks h3 {
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.frameworks .vector {
+  cursor: pointer;
+  width: 20px;
+  height: 11px;
+}
+
+.framework-list {
+  margin-bottom: 10px;
+  padding: 14px;
+  border-bottom: 1px solid #dfe1e6;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+}
+
+.button:hover {
+  background-color: #6070ff;
+  color: #fff;
+}
+
+.button:focus {
+  border: 0.0625rem solid #2230d2;
+  background-color: #2230d2;
+  color: #fff;
+}
+
+.button-enabled {
+  color: #396df2;
+  border: 0.0625rem solid #6070ff;
+}
+
+/* footer section( contact me form) */
+
+footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background: url(./images/contact\ form\ shapes-mobile1.svg) no-repeat;
+  background-color: #6070ff;
+  background-attachment: scroll;
+  background-size: contain;
+  height: 100vh;

```

Comment:
- [ ] Kindly adhere to the project requirement because as I can see you are not using CSS properly. which makes your design `inconsistent` and looks unprofessional. Please have a look at the below-attached requirement image with the design comparison table also remove `height: 100vh;`, which will help you to improve your design, my friendly suggestion to you is to always try to make your design as close as possible. 

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Design Comparison
<table>
<tr>
<th>Your Design</th>
<th>Figma Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-06-14 19-03-29](https://github.com/Fentahunn/contact_form_final/assets/66476329/761749fc-9e8f-4248-9efc-3a3000180cd2)

</td>
<td>

![Screenshot from 2023-06-06 21-01-24](https://github.com/Fentahunn/contact_form_final/assets/66476329/82cc12d7-a4ca-4498-976b-7ecdb919046b)

</td>
</tr>
</table>